exodus/restful_api/tests.py

```diff
@@ -367,7 +368,37 @@ def test_returns_one_tracker_with_categories(self):
                 'code_signature': tracker.code_signature,
                 'network_signature': tracker.network_signature,
                 'website': tracker.website,
-                'categories': ['Analytics', 'Ads']
+                'categories': ['Analytics', 'Ads'],
+                'documentation': []
+            }
+        }
+
+        response = self.client.get(self.PATH)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.json()['trackers'], expected_json)
+
+    def test_returns_one_tracker_with_documentation(self):
+        tracker = Tracker.objects.create(
+            name='Teemo',
+            description='bad tracker',
+            code_signature='com.teemo',
+            network_signature='teemo.com',
+            website='https://www.teemo.com',
+            documentation='http://link1.fr http://link2.fr/doc'
```

Comment:
Can I suggest to use `example.com` as mentioned in [section 3 of RFC 2606](https://datatracker.ietf.org/doc/html/rfc2606#section-3) for tests and documentations (same on other lines)?

Reply:
Will do, thanks!